src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -52,7 +52,27 @@ class ONNXEntryPointLowering : public OpRewritePattern<ONNXEntryPointOp> {
 namespace {
 struct FrontendToKrnlLoweringPass
     : public PassWrapper<FrontendToKrnlLoweringPass, OperationPass<ModuleOp>> {
+  // Make sure that we have a valid default constructor and copy
+  // constructor to make sure that the options are initialized properly.
+  FrontendToKrnlLoweringPass() = default;
+  FrontendToKrnlLoweringPass(const FrontendToKrnlLoweringPass &pass) {}
+
   void runOnOperation() final;
+
+public:
+  // RNN ops are lowered to other ONNX ops such as ONNXMatMulOp, ONNXSplitOp,
+  // ONNXTransposeOp, etc. These ONNX ops are then lowered into krnl ops in this
+  // pass.
+  //
+  // To write LIT tests for RNN ops, we need not to check the final generated
+  // krnl code that is lengthy but the intermediate generated code including
+  // ONNX ops. We trust the lowering of the other ONNX ops.
+  //
+  // This flag is used in LIT tests to stop the lowering of the other ONNX ops.
+  // Usage: onnx-mlir-opt --convert-onnx-to-krnl='check-rnn-ops-lowering'
```

Comment:
Thanks, that is exactly the type of super useful comments that will help us in the long run. Much appreciated